net/wireshark4/Portfile

```diff
@@ -30,12 +30,12 @@ if { ${os.platform} eq "darwin" && ${os.major} < 16 } {
     }
 }
 
-version         4.2.5
+version         4.4.0
 revision        0
 
-checksums       sha256  55e793ab87a9a73aac44336235c92cb76c52180c469b362ed3a54f26fbb1261f \
-                sha1    03293699260d2492166ac805ef0c10b8a6b531e6 \
-                size    45014156
+checksums       sha256  ead5cdcc08529a2e7ce291e01defc3b0f8831ba24c938db0762b1ebc59c71269 \
```

Comment:
Per port lint, `sha1` should no longer be used, and it should include the `rmd160` checksum type.

Reply:
@ghosthound insisted that I keep sha1 since upstream publishes sha1 checksums. See https://github.com/macports/macports-ports/pull/20477